e2e/pageobjects/ide/Ide.ts

```diff
@@ -94,10 +98,8 @@ export class Ide {
     }
 
     async clickOnNotificationButton(notificationText: string, buttonText: string) {
-        const notificationLocator: string = this.getNotificationXpathLocator(notificationText);
-        const yesButtonLocator: string = notificationLocator + `//button[text()=\'${buttonText}\']`;
-
-        await this.driverHelper.waitAndClick(By.xpath(yesButtonLocator));
+       const yesButtonLocator: string = `//div[@class='theia-notification-list-item-content-bottom']//button[text()='${buttonText}']`;
```

Comment:
As far as I understood you removed notification text from locator, so in case with several notifications we can't specify wich of them should be interacted. Am I right?

Reply:
Thx, locator was changed and checked